content/pages/index.js

```diff
@@ -23,9 +23,10 @@ import { Demo } from '../../src/components/common/Demo';
 import schema from './assets/schema.json';
 import uischema from './assets/uischema.json';
 
+const currentVersion = process.env.CURRENTVERSION;
```

Comment:
Let's show something in normal developer builds ;)
```suggestion
const currentVersion = process.env.CURRENTVERSION ?? corePackageJson.version;
```